action.yml

```diff
@@ -135,5 +135,5 @@ inputs:
     description: 'If provided, Synopsys Action will be using local network to download and execute bridge .'
     required: false
 runs:
-  using: 'node16'
+  using: 'node20'
```

Comment:
Is it tested on different OS platforms and older version of actions ?

Reply:
I have tested it on linux (cloud+self-hosted) and windows (cloud) with several type of scan. Also with different bridge version. But how can i test it with other action version since this change is only on current action version!

Reply:
I have tried with older action version but it automatically updated to latest version when runner start. There is a option to disable auto update in that case the runner says "An error occurred: Runner version v2.293.2 is deprecated and cannot receive messages.". So i think we are on the safe side.